### DISCUSSION TITLE
Fetch lemma lookup tables after packaging the application using PyInstaller

### DISCUSSION THREAD

**Discussion by BLKSerene** [explosion/spaCy#9416] [Help: Coding & Implementations] [answered]
It seems that `load_lookups` in `spacy.lookups` uses entry points to search for lemma tables, but as `PyInstaller` do not copy metadata like entry points into the frozen application, this would fail after the application is packaged. I've tried [this](https://github.com/pyinstaller/pyinstaller/issues/3050) but it did not work. Is there any other workaround?

I tried to override `spacy.lookups.load_lookups` at the beginning of my program, but it seems that it did not affect the results at all (the nlp object always uses the original `load_lookups` function).

**Comment by BLKSerene** [comment, marked as answer]
Solved by using `PyInstaller.utils.hooks.copy_metadata`.

**Reply by polm** [reply]
Thanks for sharing the solution!

**Reply by BLKSerene** [reply]
It should be noted that `__init__.py` and `about.py` must also be included in the frozen application, otherwise `importlib` would complain about not finding attributes of `spacy_lookups_data`, but I have no idea why they are necessary.
